src/root-config-lib.js

```diff
@@ -1,12 +1,18 @@
 import { registerApplication } from "single-spa";
 
 const coreApplications = {
-  "@openmrs/login": location =>
-    location.pathname.startsWith("/openmrs/login") ||
-    location.pathname.startsWith("/openmrs/spa/login"),
+  "@openmrs/login": loginActive,
   "@openmrs/devtools": () => localStorage.getItem("openmrs:devtools")
 };
 
+function routeHasPostfix(prefix, location) {
```

Comment:
I don't think `routeHasPostfix` makes a lot of sense, because we're doing `startsWith` inside of it for the url prefix. Maybe calling it `routePrefix()` instead?

Also, we should export this function from this file so that others using openmrs-esm-root-config are able to use this function.